src/cz/cvut/cognitive/distractors/CollectObjectDistraction.java

```diff
@@ -56,21 +60,33 @@
     public float distanceLeft;
     public float distanceRight;
     
+    private final AudioNode rewardSoundNode;
+    
     CollectObjectDistraction(Simulator sim, String texturePathGreen, String texturePathRed){
         COG_SCORE = 3;
         
-        
         this.sim = sim;
         this.car = sim.getCar();
         this.manager = sim.getAssetManager();
         this.camera = sim.getCamera();
-
+        
+        rewardSoundNode = new AudioNode(manager,"Sounds/FF_Victory.wav",false);
+        rewardSoundNode.setLooping(false);
+        rewardSoundNode.setPositional(false);
+        
+        rewardText = new BitmapText(manager.loadFont("Interface/Fonts/Default.fnt"), false);
```

Comment:
FIXME here and elsewhere: please use `path"+File.separator+"next` instead of `/` , it caused us some problems on other platforms (WIn?)